door43_tools/project_deployer.py

```diff
@@ -130,26 +136,24 @@ def deploy_revision_to_door43(self, build_log_key):
         for root, dirs, files in os.walk(output_dir):
             for f in sorted(files):
                 path = os.path.join(root, f)
-                key = s3_commit_key + path.replace(output_dir, '')
+                key = s3_commit_key + path.replace(output_dir, '').replace(os.path.sep, '/')
```

Comment:
Most of the time, `os.path.sep` will be `/`, except on Windows, so it would be better to just replace `\` with `/`. Otherwise we are replacing `/` with `/`. I noticed this in a few places.